Persephone/AppDelegate.swift

```diff
@@ -34,22 +31,16 @@ class AppDelegate: NSObject,
         $0.uiState
       }
     }
+    
+    instantiateControllers()
+    connectToMPDServer()
   }
 
   func connectToMPDServer() {
-    let mpdServer = App.store.state.preferencesState.mpdServer
-
-    App.mpdClient = MPDClient(
-      host: mpdServer.hostOrDefault,
-      port: mpdServer.portOrDefault,
-      withDelegate: App.mpdServerDelegate
-    )
-
-    App.mpdClient.connect()
+    App.mpdServerController.connect()
   }
 
   func instantiateControllers() {
-    _ = App.mpdServerController
     _ = App.userNotificationsController
   }
```

Comment:
Similar comment to above -- since this is now a function housing one statement, what do you think about eliminating this function?